File: DFRobot_RGBLCD/cpp/HelloWorld/HelloWorld.cpp
#include "../DFRobot_RGBLCD.h"

int main(){
  const int colorR = 255;
  const int colorG = 0;
  const int colorB = 0;
  int i = 0;
  DFRobot_RGBLCD lcd(16, 2);
  lcd.init();
  lcd.setRGB(colorR, colorG, colorB);
  lcd.print("Hello World!");
  delay(1000);
  
  while(1){
    lcd.setCursor(0, 1);
    lcd.print(i++);
    delay(1000);
  }
  return 0;
}

File: python/examples/HelloWorld.py
import sys
sys.path.append('../')
import rgb1602
import time

colorR = 255
colorG = 0
colorB = 0

lcd=rgb1602.RGB1602(16,2)

lcd.setRGB(colorR, colorG, colorB)
    
# Print a message to the LCD.
lcd.printout("hello, world!")

time.sleep(1)
i = 0

while True:
  i = i+1
  # set the cursor to column 0, line 1
  # (note: line 1 is the second row, since counting begins with 0):
  lcd.setCursor(0, 1)
  # print the number of seconds since reset:
  lcd.printout(i)
  time.sleep(1)

